11_circular_csimeone/README.md

```diff
@@ -0,0 +1,20 @@
+# Circular Distributions of CONUS Wide Streamflow
+This sub directory creates a circular plot with the total amount of water passing all gages 2 gages passing auditing requirements for 1950-2020. 
+
+# Post 1
```

Comment:
Here, you could change the header to say "the driest and wettest day of the year" to coincide with the script name and give a description of the chart that it creates. I'd also suggest linking here or specifically naming the script that creates that chart

Reply:
Added